vaadin-grid-pro-flow/src/main/resources/META-INF/resources/frontend/gridProConnector.js

```diff
@@ -1,39 +1,45 @@
-          component.focus();
-      };
+(function () {
+    const tryCatchWrapper = function (callback) {
+        return window.Vaadin.Flow.tryCatchWrapper(callback, 'Vaadin Grid Pro', 'vaadin-grid-pro-flow');
+    };
 
-      column._setEditorValue = function(editor, value) {
-          // Not needed in case of custom editor as value is set on server-side.
-          // Overridden in order to avoid blinking of the cell content.
-      }
+    window.Vaadin.Flow.gridProConnector = {
+        setEditModeRenderer: tryCatchWrapper(function (column, component) {
+            column.editModeRenderer = tryCatchWrapper(function (root) {
+                root.appendChild(component);
+                this._grid._cancelStopEdit();
+                component.focus();
+            });
 
-      column._getEditorValue = function(editor) {
-          // Not needed in case of custom editor as value is set on server-side.
-          // Overridden in order to avoid blinking of the cell content.
-          return;
-      }
-  },
+            column._setEditorValue = tryCatchWrapper(function (editor, value) {
+                // Not needed in case of custom editor as value is set on server-side.
+                // Overridden in order to avoid blinking of the cell content.
+            })
 
-  patchEditModeRenderer: function(column) {
-      column.__editModeRenderer = function(root, column, rowData) {
-          const cell = root.assignedSlot.parentNode;
-          const grid = column._grid;
+            column._getEditorValue = tryCatchWrapper(function (editor) {
+                // Not needed in case of custom editor as value is set on server-side.
+                // Overridden in order to avoid blinking of the cell content.
+                return;
+            })
+        }),
 
-          if (grid.__edited && grid.__edited.model.item.key !== rowData.item.key) {
-              grid._stopEdit();
-              return;
-          }
+        patchEditModeRenderer: tryCatchWrapper(function (column) {
+            column.__editModeRenderer = tryCatchWrapper(function (root, column, rowData) {
+                const cell = root.assignedSlot.parentNode;
+                const grid = column._grid;
 
-          const tagName = column._getEditorTagName(cell);
-          if (!root.firstElementChild || root.firstElementChild.localName.toLowerCase() !== tagName) {
-              root.innerHTML = `
-              <${tagName}></${tagName}>
-            `;
-          }
-      };
-  }
-};
+                if (grid.__edited && grid.__edited.model.item.key !== rowData.item.key) {
+                    grid._stopEdit();
+                    return;
+                }
+
+                const tagName = column._getEditorTagName(cell);
+                if (!root.firstElementChild || root.firstElementChild.localName.toLowerCase() !== tagName) {
+                    root.innerHTML = `
+                  <${tagName}></${tagName}>
+                `;
```

Comment:
Indentation is a bit wonky here. But I would prefer to make this a one liner since it's so short anyway:
```js
root.innerHTML = `<${tagName}></${tagName}>`;
```

Reply:
Updated.